Scripts/BuildUtils.ps1

```diff
@@ -0,0 +1,71 @@
+$DefaultAccountName = "mirasrael"
+$ProjectSlug = "runtimegraphicsinfo"
+$RuntimesPath = "$PSScriptRoot\..\Runtime\Plugins"
+
+function BuildUtils-DownloadAppVeyorRuntimes([string]$Token = "", [string]$AccountName = "", [string]$BuildId = "") {
+    $apiUrl = 'https://ci.appveyor.com/api'
+
+    if ($Token -eq "")
+    {
+        $tokenFilePath = "$HOME/.appveyor/token"
+        if ([System.IO.File]::Exists($tokenFilePath))
+        {
+            $Token = Get-Content -Path $tokenFilePath
+        }
+        else
+        {
+            Write-Host "AppVeyor token not found";
+            Exit 1;
+        }
+    }
+
+    if ($AccountName -eq "")
+    {
+        $AccountName = $DefaultAccountName
+    }
+
+    $token = $Token
+    $headers = @{
+        "Authorization" = "Bearer $token"
+        "Content-type" = "application/json"
+    }
+
+    if ($BuildId -eq "")
+    {
+        # get project with last build details
+        $project = Invoke-RestMethod -Method Get -Uri "$apiUrl/projects/$AccountName/$ProjectSlug" -Headers $headers
+    }
+    else
+    {
+        # get project with specific build details
+        $project = Invoke-RestMethod -Method Get -Uri "$apiUrl/projects/$AccountName/$ProjectSlug/build/$BuildId" -Headers $headers
+    }
+
+    # download artifacts for all jobs in the build
+    foreach ($job in $project.build.jobs)
+    {
+        $jobId = $job.jobId
+
+        # get job artifacts (just to see what we've got)
+        $artifacts = Invoke-RestMethod -Method Get -Uri "$apiUrl/buildjobs/$jobId/artifacts" -Headers $headers
+
+        foreach ($artifact in $artifacts)
+        {
+            $artifactFileName = $artifact.fileName
+            $libName = Split-Path -Path $artifactFileName -Leaf
+            $runtime = switch ($job.name)
+            {
+                "Image: Visual Studio 2019; Platform: x86" { "x86" }
```

Comment:
Could we change these to runtime identifier style (win-x64 etc)?

Reply:
No, afaik. It is auto-generated name from AppVeyor.

Reply:
I mean the right-hand side, x86 vs x86-64, which is the output directory... I don't think it matters to unity because we're using asmdefs. Doesn't really matter but would be nice to have it consistent with builds.